lib/generators/boring/sidekiq/install/install_generator.rb

```diff
@@ -62,6 +67,17 @@ def add_sidekiq_routes
                          after: "Rails.application.routes.draw do\n"
       end
 
+      def add_sidekiq_worker_to_procfile
+        return if options[:skip_adding_sidekiq_to_procfile]
+
+        if File.exist?("Procfile.dev")
+          say "Adding sidekiq worker to Procfile.dev", :green
+          append_to_file "Procfile.dev", "worker: bundle exec sidekiq"
+        else
+          say "Procfile.dev not found. Skipping adding sidekiq worker to Procfile.dev", :yellow
```

Comment:
It's fine to return early if File is not found, we don't need to show file not found message